igvc_navigation/launch/set_waypoint_file_path.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <arg name="file_path"/>
+    <param name="/waypoint_file_path" value="$(arg file_path)"/>
+</launch>
```

Comment:
For ease of use, I think it would be nice to be able to split this into like a "folder" and a "filename", so that instead of having to specify the entire path on the command line, we can specify just the name of the `.csv`